fuel/transformers/__init__.py

```diff
@@ -97,8 +114,9 @@ class Mapping(Transformer):
         data under source names `add_sources`.
 
     """
-    def __init__(self, data_stream, mapping, add_sources=None):
-        super(Mapping, self).__init__(data_stream)
+    def __init__(self, data_stream, produces_examples, mapping,
```

Comment:
Although we could simply call the superclass' constructor by passing `self.data_stream.produces_examples`. An error will be raised if someone tries to pass `produces_examples` as a kwarg.